config.go

```diff
@@ -27,7 +27,7 @@ type RuntimeConfig interface {
 	//
 	// Here are the notable effects:
 	// * Adds `memory.fill`, `memory.init`, `memory.copy` and `data.drop` instructions.
-	// * Adds `table.fill`, `table.init`, `table.copy` and `elem.drop` instructions.
```

Comment:
table.init is actually depending on the reference type (as it takes reference typed value as an argument) so essentially in ReferenceType proposal

Reply:
maybe the feature should say it is dependent on another (ex enabling this enables the other, and visa versa, disabling retypes disables this as it is not functional without it)